src/core/android/SDL_android.c

```diff
@@ -2766,4 +2769,85 @@ int Android_JNI_OpenURL(const char *url)
     return ret;
 }
 
+static jstring Android_GetExceptionString(JNIEnv *env)
+{
+    jthrowable throwable = (*env)->ExceptionOccurred(env);
+
+    if (throwable)
+    {
```

Comment:
SDL style is `if (x) {`

Reply:
Whoops, missed this one. Although I might just remove this whole function altogether as per below review.